### PR TITLE
Default the managerhosts value to 127.0.0.1

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

File: src/agent/agent.c [hunk 1]
```diff
@@ -516,10 +516,12 @@ bool agent_parse_config(Agent *agent, const char *configfile) {
         }
 
         value = cfg_get_value(agent->config, CFG_MANAGER_HOST);
-        if (value) {
-                if (!agent_set_host(agent, value)) {
-                        return false;
-                }
+        if (!value) {
+                value="127.0.0.1";
```
Comment (engelmi) [inline src/agent/agent.c]: I think the value of the default host should defined via `#define HIRTE_DEFAULT_HOST "127.0.0.1"` next to the default port in [protocol.h](https://github.com/containers/hirte/blob/main/src/libhirte/common/protocol.h#L6).

File: doc/man/hirte-agent.conf.5.md [hunk 2]
```diff
@@ -31,8 +31,7 @@ options. The option doesn't have a default value.
 
 #### **ManagerHost** (string)
 
-The host used by `hirte-agent` to connect to `hirte`. Must be a valid IPv4 or IPv6. The option doesn't have a default
-value, it's mandatory to set this option for each hirte-agent.
+The host used by `hirte-agent` to connect to `hirte`. Must be a valid IPv4 or IPv6. The option defaults to 127.0.0.1, it's mandatory to set this option on remote hirte-agents.
```
Comment (mwperina) [inline doc/man/hirte-agent.conf.5.md]: Could you please update the default value also in below files?

https://github.com/containers/hirte/blob/main/config/agent/agent.conf#L14
https://github.com/containers/hirte/blob/main/config/agent/hirte-default.conf#L10
https://github.com/containers/hirte/blob/main/doc/man/hirte-agent.1.md?plain=1#L27

**Review by mwperina — CHANGES_REQUESTED** [review]

File: src/agent/agent.c [hunk 3]
```diff
@@ -516,10 +516,12 @@ bool agent_parse_config(Agent *agent, const char *configfile) {
         }
 
         value = cfg_get_value(agent->config, CFG_MANAGER_HOST);
-        if (value) {
-                if (!agent_set_host(agent, value)) {
-                        return false;
-                }
+        if (!value) {
```
Comment (engelmi) [inline src/agent/agent.c]: The default port is set in the [agent_new()](https://github.com/containers/hirte/blob/main/src/agent/agent.c#L323) and - if I understand it correctly - overridden if it is defined in any of the config options. For consistency, it should be the same for the host.

**Review by mwperina — APPROVED** [review]
+1

**Comment by rhatdan** [comment]
@engelmi PTAL and merge if it you will.

**Review by engelmi — APPROVED** [review]
LGTM

**Pull request merged**